R/simulate.r

```diff
@@ -46,47 +56,57 @@
 #'
 #' See References below for some literature on the subject.
 #'
-#' ## Specifying `serials_dist` in `simulate_tree()`
+#' ## Specifying `serials_dist`
 #'
 #' `serials_dist` must be specified as a named or
-#' [anonymous/inline/unnamed function](https://en.wikipedia.org/wiki/Anonymous_function#R) # nolint
+#' [anonymous/inline/unnamed function](https://en.wikipedia.org/wiki/Anonymous_function#R) #nolint
```

Comment:
I don't think you can use `nolint` in comments - this gets rendered into the man page.

Reply:
It's strange because it was suggested in https://github.com/r-lib/lintr/issues/379 yet when I implement it, it gets rendered.

Reply:
I think the suggestion there is to have a line with `#nolint start` before the whole roxygen block and one with `#nolint end` at the end